packages/tools/src/stateManagement/segmentation/helpers/computeStackLabelmapFromVolume.ts

```diff
@@ -29,6 +29,10 @@ export async function convertVolumeToStackSegmentation({
 }): Promise<void> {
   const segmentation = getSegmentation(segmentationId);
 
+  if (!segmentation) {
+    return;
+  }
+
   const { volumeId } = segmentation.representationData
     .Labelmap as LabelmapSegmentationDataVolume;
   const segmentationVolume = cache.getVolume(volumeId) as Types.IImageVolume;
```

Comment:
If you just return updateStackSegmentationState then you don't need the async declaration and the function becomes a bit simpler.